.circleci/config.yml

```diff
@@ -189,7 +189,7 @@ jobs:
 
       - run:
           name: generate swagger documentation
-          command: bundle exec rake rswag:specs:swaggerize
+          command: bundle exec rspec spec/swagger --format Rswag::Specs::SwaggerFormatter --order defined
```

Comment:
I'm wondering if the same swagger specs could be excluded from the test job? Or maybe it's worth running them again anyway 🤔